rasa/shared/core/domain.py

```diff
@@ -763,6 +775,7 @@ def _get_active_loop_sub_state(
             tracker: dialog state tracker containing the dialog so far
         Returns:
             a dictionary mapping "name" to active loop name if present
+
         """
 
```

Comment:
any idea why the check doesn't fail at this line with "D202 No blank lines allowed after function docstring" ?

Reply:
It now works. Some validations seem a bit fragile. In this case the empty line after the first line of the docstring was missing.